test/qa-tests/jstests/export/force_table_scan.js

```diff
@@ -58,12 +58,10 @@
     } else {
       assert.eq(true, queries[0].command.snapshot || queries[0].command.hint._id === 1);
     }
-  } else {
```

Comment:
I feel like this change obscures the intended code flow, but the linter requires it :(